nasa/earthdata-search
# EDSC-3144: Adding CSDA modal

# Overview

### What is the feature?

Adds a modal for CSDA collections

### What is the Solution?

Add a more details link to the CSDA collection information, and trigger a modal when clicked

### What areas of the application does this impact?

Granule results page

# Testing

### Reproduction steps

- **Environment for testing:** UAT
- **Collection to test with:** C1240538039-CSDA

1. Visit the granule results page
2. Click the more details button
3. Confirm the modal with CSDA information is displayed

# Checklist
